ap2-receiver.py

```diff
@@ -440,6 +441,12 @@ def do_TEARDOWN(self):
         self.send_header("Server", self.version_string())
         self.send_header("CSeq", self.headers["CSeq"])
         self.end_headers()
+        
+        # Erase the hap() instance, otherwise reconnects fail
+        self.server.hap = None
+
+        # terminate the forked event_proc, otherwise a zombie process consumes 100% cpu
+        self.event_proc.terminate()
```

Comment:
This seems to cause the tcp connection to be closed. Hence when skipping a track forward airplay disconnects.

Reply:
Well, hanging processes seem worse than disconnects at fast forward. But this is a good find. Do your instincts have any proposals about how to reconcile the CPU fix and a disconnect fix?